src/libsyntax/feature_gate.rs

```diff
@@ -533,6 +530,8 @@ declare_features! (
     // Allows the sysV64 ABI to be specified on all platforms
     // instead of just the platforms on which it is the C ABI
     (accepted, abi_sysv64, "1.24.0", Some(36167)),
+    // Allows `repr(align(u16))` struct attribute (RFC 1358)
```

Comment:
This should say `16` not `u16`.

Reply:
Ah yes, my intention was to specify the type of the alignment parameter, but it's totally unclear here and no longer u16 anyway.